server/src/providers/blockchain.ts

```diff
@@ -122,17 +122,25 @@ export class blockchain {
    * Initializing web3 instances and all required contracts
    */
   async init () {
-    const props = await PropertyProvider.getAll()
-    log.debug('Config/Status:', props)
-    if(conf.reset && conf.reset != props.lastVersion)
-    {
-      log.info("reseting database", {version: conf.reset, lastVersion: props.lastVersion })
-      await Promise.all([PropertyProvider.model.deleteMany({}),
-      walletsProvider.model.deleteMany({}),
-      AboutClaimTransactionProvider.model.deleteMany({}),
-      AboutTransactionProvider.model.deleteMany({}),
-      AddressesClaimedProvider.model.deleteMany({})])
-      PropertyProvider.set("lastVersion", conf.reset)
+    const lastVersion = await PropertyProvider.get('lastVersion')
+
+    log.debug('LastVersion value:', {
+      lastVersion,
+      reset: conf.reset,
+    })
+
+    if(conf.reset && Number(conf.reset) != Number(lastVersion)) {
```

Comment:
1. Then we cant pass null/undefined (falsy) value as default - it will fail.
2. We can't do it. The Property model is a universal model where we can store property names and their values. The value could be a number as well as a string. So we need to convert it manually

Reply:
@yaroslav-fedyshyn-nordwhale i've pushed the fix. will test dashboard now locally